kbmd/kbmd.c

```diff
@@ -41,8 +41,6 @@
 #include <unistd.h>
 #include "kbmd.h"
 
-bunyan_logger_t *blog;
```

Comment:
Can you explain to me what's going on here?

Reply:
Yes.  It's already declared elsewhere:
```
kebe(~/ws/kbmd)[0]% git grep "bunyan_logger_t \*blog"
common/common.c:bunyan_logger_t *blog;
common/common.h:extern bunyan_logger_t *blog;
kbmd/kbmd.c:bunyan_logger_t *blog;
kebe(~/ws/kbmd)[0]% git checkout OS-8326
Switched to branch 'OS-8326'
Your branch is up to date with 'origin/OS-8326'.
kebe(~/ws/kbmd)[0]% git grep "bunyan_logger_t \*blog"
common/common.c:bunyan_logger_t *blog;
common/common.h:extern bunyan_logger_t *blog;
kebe(~/ws/kbmd)[0]% 
```